### DISCUSSION TITLE
How do you check language module versions?

### DISCUSSION THREAD

**Discussion by MagnusOxlund** [nginx/unit#1449] [General]
Once a language module is installed, how do you check the exact versions it supports?

**Comment by MagnusOxlund** [comment]
For the PHP language module:

`sudo cat /var/log/unit/unit.log | grep php`

**Comment by ac000** [comment]
If you have Unit 1.33 or newer, you can use the `/status/modules` endpoint. E.g.

```
$ curl --unix-socket /opt/unit/control.unit.sock http://localhost/status/modules/
{
	"wasm": {
		"version": "0.1",
		"lib": "/opt/unit/modules/wasm.unit.so"
	}
}
```
